lib/kernel/doc/src/kernel_app.xml

```diff
@@ -194,6 +194,12 @@
             <c>logger:set_primary_config(level, Level)</c></seealso>.</p>
 	<p>Defaults to <c>notice</c>.</p>
       </item>
+      <tag><marker id="logger_default_metadata"/><c>logger_default_metadata = Metadata</c></tag>
+      <item>
+	<p>Specifies default value of process metadata for log events.</p>
+	<p><c>Metadata = map()</c></p>
+	<p>Defaults to <c>#{}</c>.</p>
+      </item>
```

Comment:
<c>logger_chapter.xml</c> also should be updated with documentation regarding this option.